crates/task-impls/src/network.rs

```diff
@@ -292,6 +303,20 @@ impl<
                         TransmitType::Direct(membership.leader(vote.view_number() + 1)),
                     )
                 }
+                HotShotEvent::QuorumProposalRequestSend(view_number, sender_key) => (
+                    sender_key.clone(),
+                    MessageKind::<TYPES>::from_consensus_message(SequencingMessage::General(
+                        GeneralConsensusMessage::ProposalRequested(view_number, sender_key),
+                    )),
+                    TransmitType::Direct(membership.leader(view_number)),
```

Comment:
Just FYI the next task related to this is to make it so we can request proposals from the DA committee as well so this logic will have to get significantly more complex.  Basically the logic of do_vid in the request task will need to live here now, or we can just handle this event there like before